types/schema/data/Proposal.ts

```diff
@@ -104,13 +109,21 @@ export interface BaseProposal {
   newDwellings?: {
     newBuild?: {count: number};
   };
+  units?: {
+    residential: {
+      development: DevelopmentType;
+      type: UKResidentialUnitType;
+      bedrooms: number;
+      identicalUnits: number;
+    }[];
+  };
 }
 
 /**
  * @id #LondonProposal
  * @description Proposal details for project sites within the Greater London Authority (GLA) area
  */
-export interface LondonProposal extends BaseProposal {
+export interface LondonProposal extends Omit<BaseProposal, 'units'> {
```

Comment:
`units` has an entirely different shape depending if the site is in `London` or the wider UK, so `Omit` allows us to essentially ignore/overwrite the UK definition when extending the interface for London.

There's other similar variables we're going to need to do this for soon too like `parking`.

Reply:
This is a neat solution and maintains the relationship between the two `Proposal` types to pick up any future changes that will affect both 👍